syntax = "proto3";

package observability;

option go_package = "github.com/minhtt159/tf-telemetry/internal/gen/pb;pb";

enum Platform {
  PLATFORM_UNKNOWN = 0;
  WEB = 1;
  IOS = 2;
  ANDROID = 3;
  OTHER = 4;
}

enum NetworkType {
  NET_UNKNOWN = 0;
  NET_WIFI = 1;
  NET_CELLULAR_5G = 2;
  NET_CELLULAR_4G = 3;
  NET_CELLULAR_3G = 4;
  NET_CELLULAR_2G = 5;
  NET_WIRED = 6;
  NET_OFFLINE = 7;
}
